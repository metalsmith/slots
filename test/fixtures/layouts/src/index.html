---
title: Nothing else matters
layout: default.njk
debug: true
---
Hello world!

---
slot: header
---
header-content

---
slot: footer
---
footer-content
