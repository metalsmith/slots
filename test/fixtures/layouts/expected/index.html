<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Nothing else matters</title>
</head>
<body>
  <header>
    <span class="debug">header</span>
    header-content
  </header>
  <h1>Nothing else matters</h1>
  Hello world!
  <footer>
    <span class="debug">footer</span>
    footer-content
  </footer>
</body>
</html>